development/proposal_template.md

```diff
@@ -0,0 +1,42 @@
+# Proposal Template
+## Executive Summary
+### The Current Situation
+Briefly describe the client’s most important issues, followed by the focusing
+question. This gives you the opportunity to show that you understand the
+business environment, the specific issues, and the most important question your client would like answered. Be brief and to the point.
+
+Focusing Question – Check and agree on the overall question your client
+wants answered. Take great care to get this right as it sets the direction for your whole investigation.
+A focusing question has 7 parts, an opening, a purpose (why), an act (what), an agent (who), an agency (how), a scene (where) and a qualification (standard).
+
+### The Ideal Situation
+Describe the most important targets the client would like to achieve. This is an opportunity to show alignment with business goals.
+
+- Business goals - Describe the major business goals that your project will help the organisation to fufill
+- Project goals - The goals must clearly show what the investigation will achieve. What is your project doing?
+
+### The Proposed Solution
+Outline your proposal from a high level and the benefits it will deliver. Here you must show that you have a workable solution that delivers value, and that you have the technical ability to deliver the solution to your client.
+### Project Management and Costs
+Briefly outline the proposed size and skills of the team, plus the timing, costs, and co-commitments. Provide a concise appraisal of the resources, the timing and any co-commitments needed for successful implementation of the project. Graphs are very useful in this section.
+
+## Scope of Work
+This section is the heart of your proposal and should be accessible to the layman. Avoid technical terminology and use diagrams, charts and tables to express complex points clearly. Expand on the points from the executive summary.
+### The Current Situation
+Deepdive the points made in the current situation above. Provide additional context and describe the problem more completely.
+
+
+### The Ideal Situation
+Describe the 'why' 'what' and 'how'. Why is this important? What will the outcome be? 
+
+### The Proposed Solution
+This is your approach in detail, still free of technical terms and accessible still to the layman. This is where you introduce your solution and describe why your approach is the best available. Make sure to note any relevant costs, benefits and risks. The aim is to describe the value you will deliver using your approach.
+
+### Managing the Project
```

Comment:
We don't want people considering people and budget.  We want them to consider timelines of major pieces, but it's probably too much to ask for more.